app/models/api/selfEmploymentBusiness/createAmendSEAnnualSubmission/AnnualAdjustments.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.api.selfEmploymentBusiness.createAmendSEAnnualSubmission
```

Comment:
it's quite a long package name, can it be a bit flatter, why not keep everything in api

Reply:
I think currently having everything in api is a little confusing personally. I would like to see which api the model i am looking at is referring to, as in main currently there is `BusinessData` model and a `AllSelfEmploymentData` model. It is not obvious that `BusinessData` is modelling the `business details api`